capsule.bxb

```diff
@@ -2,9 +2,6 @@ capsule {
   id (example.spaceResorts)
   version (1.0.0)
   format (3)
-  runtime-flags {
-    support-halt-effect-in-computed-inputs
-  }
```

Comment:
This block was duplicated in this file (see line 11 below)

Reply:
Didn’t notice, a git merge artifact. Thanks for catching